src/lib/install.sh

```diff
@@ -5,8 +5,9 @@ import assert
 
 _install_name_='install.sh'
 
-#: $1 - path to source, $2 - name, $3 - version, $4 (optional) - relative path to binary (w/o the first dir)
 install_single() {
+    #: Used to install single-binary recipes
+    #: $1 - path to source, $2 - name, $3 - version, $4 (optional) - relative path to binary (w/o the first dir)
     echo "install_single($1, $2, $3, $4) called!"
```

Comment:
(Not part of this change but still necessary:) Please rewrite this and similar `echo`-based log statements using the logging module functions (like `_debug`, `_info`, etc.)